docs/src/_guide/advanced-topics.md

```diff
@@ -13,3 +13,159 @@ Most of Meltano's features are available without installing any additional packa
 * `azure` - Support for using Azure Blob Storage as a [state backend](/concepts/state_backends)
+
+## Airbyte Connector Integration FAQ
+
+### How do the Singer and Airbyte specifications relate?
+
+The Singer specification was started in 2016 by Stitch Data. It specified a data transfer format that would allow any number of data systems, called taps, to send data to any data destination, called targets. Airbyte was incorporated in 2020 and created their own specification that was heavily inspired by Singer. There are differences but the core of each specification is sending new-line delimited JSON data from STDOUT of a tap to STDIN of a target.
+
+### How does the Airbyte Connector with Meltano integration work?
+
+A community member used the Meltano Singer SDK to write tap-airbyte-wrapper. This wrapper connector calls the Docker image for a given Airbyte Source and translates the messages into a Singer-compatible format. The output of the tap is conformed to the Singer standard and then can be sent to any Singer target, many of which are listed on [MeltanoHub](hub.meltano.com/loaders/).
+
+### How do I get help if I need it?
+
+We first recommend you read through this FAQ to see if your question has been answered. We next recommend searching the [Meltano docs](https://docs.meltano.com) to see if your answer is there.
+
+If you still need help we’d recommend either filing an issue in the [Meltano Github repository](https://github.com/meltano/meltano/issues) or join our [Slack community](https://meltano.com/slack) to ask there.
+
+We also have [weekly Office Hours](https://www.addevent.com/calendar/Li390615) where you can talk to the Meltano team and ask any questions you have to us!
+
+### Does this wrap the Airbyte UI or API?
+
+No. This integration only makes it possible to run Airbyte [Source Connectors](https://docs.airbyte.com/category/sources) within your Meltano project.
+
+### Does this support Airbyte destination connectors as well?
+
+Airbyte destinations are _not_ supported with tap-airbyte-wrapper. This is a feature that could be added in the future.
+
+### What is the recommended way to install and use these connectors?
+
+Any connector that is listed on MeltanoHub as being maintained by `airbyte` can be installed and run as you would any other connector. Use `meltano add extractor &lt;tap>` to add and install.
+
+### What are the limitations of running Airbyte-based connectors with Meltano?
+
+Based on the current implementation and testing, the main challenge is putting these connectors into production with Meltano. See the next question for more details on this.
+
+### Am I able to put my Meltano project with Airbyte connectors in production?
+
+Yes, assuming you can set up Docker-in-docker. Since each Airbyte connector is itself a Docker image, running inside a dockerized Meltano project would require “docker in docker” which has some possible challenges around privileges and permissions on certain systems.
```

Comment:
This assumes that production deployments require Docker which isnt necessarily true. We might want to consider prefacing that the challenges only come with using Dockerized meltano projects. If someone doenst package with a docker image and the production machine has docker available then this just works.

Reply:
Clarified. Feel free to make direct edits if you feel there is a better way to say it.